cmd/networkcmd/start.go

```diff
@@ -30,7 +30,7 @@ is already running or if no subnets have been deployed.`,
 }
 
 func startNetwork(cmd *cobra.Command, args []string) error {
-	sd := subnet.NewLocalDeployer(app)
+	sd := subnet.NewLocalDeployer(*app)
```

Comment:
No action to be take for this PR, but I created a ticket https://ava-labs.atlassian.net/browse/AV-1857. Network commands should not have a need to use functionality of a subnet deployer object.

Reply:
that's a good call